jose-jwt/jose-jwt.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Seems like it's no longer possible to build project without .csproj file on newest dotnet cli?